README.md

```diff
@@ -1,94 +1,79 @@
-This was the result of a discovery sprint, I am in the process of cleaning it up for easier local install.
-
 ## Local set up
 
-This is using Pipenv and Postgres
-
-### Pipenv info
-
- https://github.com/pypa/pipenv
-
-### Postgres
-
- https://www.postgresql.org/download/
-
-I recommend [pgcli](https://www.pgcli.com/) for interfacing with postgres
+Install Docker
 
+    https://www.docker.com/get-started
 
-In pgcli or your sql shell command to create datebase with:
 
-    create database crt_portal;
+Use a long, random string for your secret key and save it in your local environment.
 
+    export SECRET_KEY=''
```

Comment:
A nice little developer experience improvement would be to generate and set a random SECRET_KEY automatically during local development — @LindsayYoung give this comment a 👍react if you want me to make an issue for that.

Reply:
I thought about it but thought it is some thing we can add later. Feel free to make a ticket!

Reply:
Addressed in #13, it's just one line of code so I figured a PR was easier than a ticket.

Reply:
I just pushed a change to generate a random default to make the setting optional.

Reply:
OK!